src/app/address/controllers/address.js

```diff
@@ -3,23 +3,45 @@ const BaseController = require("hmpo-form-wizard").Controller;
 class AddressController extends BaseController {
   locals(req, res, callback) {
     super.locals(req, res, (err, locals) => {
-      locals.addressPostcode = req.sessionModel.get("addressPostcode");
-      const address = req.sessionModel.get("chosenAddress");
+      const addresses = req.sessionModel.get("addresses");
+      let address;
+
```

Comment:
Opted for the originalURL but I'm wondering if there is a nicer way to know I'm in the previous address journey and I'm specifically editing a step

Reply:
I think there is, but I'm not sure off the top of my head...